server/sonar-server/src/main/java/org/sonar/server/branch/ws/ListAction.java

```diff
@@ -103,6 +104,7 @@ public void handle(Request request, Response response) throws Exception {
       Map<String, LiveMeasureDto> qualityGateMeasuresByComponentUuids = dbClient.liveMeasureDao()
         .selectByComponentUuidsAndMetricKeys(dbSession, branches.stream().map(BranchDto::getUuid).collect(toList()), singletonList(ALERT_STATUS_KEY))
         .stream().collect(uniqueIndex(LiveMeasureDto::getComponentUuid));
+      // TODO perhaps need to do similarly for PULL_REQUEST too?
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Complete the task associated to this TODO comment. ([squid:S1135](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1135))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=feature%2Fjanos%2FSONAR-10365%2Fadd-pull-request-type&issues=AWF7Yi5UGUleEzPZFx90&open=AWF7Yi5UGUleEzPZFx90)

[ISSUE_KEY]: AWF7Yi5UGUleEzPZFx90

Reply:
I already did a first implementation and it's coming very soon (with the last changes). As discussed, it will be in a separated WS so this WS will NOT return pull requests